README.md

```diff
@@ -7,6 +7,106 @@
 
 This is a desktop application acting as a Gradle Tooling API client.
 
+## Usage
```

Comment:
💭 Maybe a sentence or two at the top that describes what this project is for.

Reply:
"This is a desktop application acting as a Gradle Tooling API client."

I don't think this is very descriptive or helpful.  What does it demonstrate?  Why does it use the TAPI?  Things like that.

Reply:
I added some words, PTAL

Reply:
Looks good!